_includes/releases/v23.1/v23.1.0-rc.1.md

```diff
@@ -0,0 +1,78 @@
+## v23.1.0-rc.1
+
+Release Date: May 2, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-0-rc-1-sql-language-changes">SQL language changes</h3>
+
+- The [`crdb_internal.transaction_contention_events` job](../v23.1/crdb-internal.html) now succeeds even if an error occurs getting the name of a non-SQL key. [#101871][#101871]
+- The new SQL activity updater [job](../v23.1/admin-ui-jobs-page.html) updates the system tables `transaction_activity` and `statement_activity` to aggregate the top 500 of each of the 6 columns of the `statistics` tables. This improves the performance of the user interface. [#101995][#101995]
```

Comment:
It's not clear what "SQL activity updater" is. Does it refer to `sql.stats.activity.persisted_rows.max` in the PR?

Reply:
No -- I didn't properly understand the change when I drafted this. I've made further edits now.